### DISCUSSION TITLE
Contributor call: 8 July 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4382] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: July 8, 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jul-8-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
👀

**Comment by SeaZeeZee** [comment]
UI and visual accesibility

**Comment by tangerineshark** [comment, marked as answer]
### Contributor Call Notes
(Sorry these notes are late, that's on @tangerineshark , as are any inaccuracies or misrepresentations)

Attendees: @BenJamesBen , @bobgeorgethe3rd , Thrunic, @SeaZeeZee , Braveness23, @lexander2002 , @eriese , @tangerineshark 

@BenJamesBen is working on a test garment of Unice and Jane, but his machine broke on Unice and so he's completing Jane by hand. Ran into some trouble with seam allowance and ease.

Braveness23 took a vacation to Canada, took a laptop along, started making a design, having a lot of fun fumbling through. I thought @joostdecock would be charmed by the statement: "community's been fantastic helping me out - I love it, I absolutely love it."
- Working on an overalls pattern that is neither farmer nor construction worker-esque
- also interested in working on a hernia support option
- @bobgeorgethe3rd , @SeaZeeZee , and Thrunic offered to help as needed

UI and visual accessibility
- @SeaZeeZee went through theme(s) to make sure colors have enough contrast
- As a collective, we have insufficient experience to make sure that things work for colorblind folks, Thrunic offered to support
- @SeaZeeZee 's goal is to make light/dark themes as accessible as possible, and then maybe also make a minimal theme (could also function as a high-contrast theme?)
- We could use some folks who are willing to test the new themes, particularly for accessibility if you're colorblind and/or use screenreaders (esp. hot keys) (@tangerineshark might be able to help with this - almost def with the former, and possibly with the latter)

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
